pkg/clusteragent/admission/mutate/autoinstrumentation/language_versions.go

```diff
@@ -101,12 +101,17 @@ var supportedLanguages = []language{
 	python,
 	dotnet,
 	ruby,
+	php, // PHP only works with injection v2, no environment variables are set in any case
 }
 
 func (l language) isSupported() bool {
 	return slices.Contains(supportedLanguages, l)
 }
 
+func (l language) isEnabledByDefault() bool {
+	return l != "php"
+}
+
```

Comment:
Question:

Why is PHP not enabled by default? How is it different from other supported languages?

Reply:
PHP was added later than the other languages, so it will remain in Preview when SSI goes GA.